src/test/java/reposense/util/FileUtilTest.java

```diff
@@ -0,0 +1,60 @@
+package reposense.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileUtilTest {
+
+    private static final Path FILE_UTIL_TEST_DIRECTORY = new File(FileUtilTest.class.getClassLoader()
+            .getResource("FileUtilTest").getFile()).toPath().toAbsolutePath();
+    private static final Path ARCHIVE_ZIP_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
+            FileUtil.ZIP_FILE);
```

Comment:
In cases like this, try to insert the line break at a higher node of the syntax tree, to improve readability. e.g.,
```java
private static final Path ARCHIVE_ZIP_PATH = 
        Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(), FileUtil.ZIP_FILE);
```